README.md

```diff
@@ -1,5 +1,5 @@
 # VectorDB-recipes
-Example Applications
+Examples
 
```

Comment:
Would be good to add a short description about how these examples are for LanceDB, to get started visit docs, etc.

Reply:
Yeah good point.

Reply:
actually, let's not add lancedb at the top as it'll seem marketing-y..We can add it at the bottom maybe?